lib/logstash/inputs/http_poller.rb

```diff
@@ -42,6 +54,20 @@ def register
 
     @logger.info("Registering http_poller Input", :type => @type, :schedule => @schedule, :timeout => @timeout)
 
+    @url_field = ecs_select[disabled: "[#{metadata_target}][request][url]", v1: "[url][full]"]
+    @method_field = ecs_select[disabled: "[#{metadata_target}][request][method]", v1: "[http][request][method]"]
+    @host_field = ecs_select[disabled: "[#{metadata_target}][host]", v1: "[host][hostname]"]
+    @response_code_field = ecs_select[disabled: "[#{metadata_target}][code]", v1: "[http][response][status][code]"]
```

Comment:
the ECS field name is [`http.response.status_code`](https://github.com/elastic/ecs/blob/1.11/generated/ecs/ecs_flat.yml#L4510) so we should use `[http][response][status_code]`

Reply:
oh, good catch! my eyes somehow see it as `status.code` 🤦